### PR TITLE
Fix ContinuousRectangleBorder bottom radius

### PR DESCRIPTION
**Pull request by a9udn9u**
## Description

Currently bottom left radius is drawn on bottom right, and bottom right radius is drawn on bottom left. This commit fixes the behavior.

## Related Issues

None.

## Tests

I updated the existing test case, it used to set bottom right radius but check against an image with bottom left round corner, now the test set bottom left radius.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

**Review by goderbauer — COMMENTED** [review]
Good catch!

Could you please also add a test that verifies that "topRight" and "bottomRight" draw the right thing?

File: packages/flutter/test/painting/continous_rectangle_border_test.dart [hunk 1]
```diff
@@ -82,7 +82,7 @@ void main() {
         shape: const ContinuousRectangleBorder(
           borderRadius: BorderRadius.only(
             topLeft: Radius.circular(28.0),
-            bottomRight: Radius.circular(14.0),
+            bottomLeft: Radius.circular(14.0),
```
Comment (goderbauer) [inline packages/flutter/test/painting/continous_rectangle_border_test.dart]: For future reference: This is what the golden file looks like: https://flutter-gold.skia.org/img/images/9404cb8a76e0c694926a2b6cdb08181a.png

Comment (a9udn9u) [reply packages/flutter/test/painting/continous_rectangle_border_test.dart]: How do you generate images like this? I want add different radius on all four corners so we can verify them all in one go.

Comment (Piinks) [reply packages/flutter/test/painting/continous_rectangle_border_test.dart]: Hi @a9udn9u, do you mean you want to add more image tests as part of this change? I can help with that. Visit our wiki [Writing a golden file test for package:flutter](https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package:flutter) and let me know if that sounds like what you are looking for. :)

**Comment by goderbauer** [comment]
@a9udn9u Do you have plans to come back to this PR?

**Comment by a9udn9u** [comment]
Yes but unfortunately I don't have time right now, it would be great if someone can take over otherwise I might be able to work on this in mid-Dec.

**Comment by goderbauer** [comment]
@Piinks Are you interested in taking this on?

**Comment by Piinks** [comment]
I can pick it up. 😉 I will close this to reopen as a new PR.

**Pull request closed without merging**